### ISSUE TITLE
Pycharm Q plugin suggestions within python debug console

### ISSUE THREAD

**Issue by tomas-sk** [aws/aws-toolkit-jetbrains#5220]
**Describe the bug**
Q makes annoying suggestions within python debug console. It is more of a barrier than help. Suggested code has never any value, large blocks with function definitions is not what u usually do in debug console.

**To reproduce**

**Expected behavior**

**Screenshots**
<img width="1109" alt="image" src="https://github.com/user-attachments/assets/e01a3b05-9fbd-4280-a96b-f7b265563924" />

**Your Environment**
- OS: Win 11
- JetBrains product: PyCharm 
- JetBrains product version: 2024.3.1.1 (Professional Edition)
- AWS Toolkit version:
- SAM CLI version:
- JVM/Python version: 3.11

**Comment by nicovong** [comment]
I totally agree, it is so annoying because it expands the input zone and hide the previous outputs

![Image](https://github.com/user-attachments/assets/ec2865e1-2a1b-4638-b8e3-733aa4955046)

**Comment by paulbayer** [comment]
+1 - It's always a useless suggestion in the Python console or debug. It's not context-aware, so please make it stop!!
